frictionless/server/file/list.py

```diff
@@ -16,5 +16,14 @@ class Result(BaseModel):
 @router.post("/file/list")
 def server_file_list(request: Request, props: Props) -> Result:
     project: Project = request.app.get_project(props.session)
+    records = project.database.list_records()
     items = project.list_files()
+    for index, item in enumerate(items):
```

Comment:
This logic needs to go to `project.list_files` as we don't use endpoints for any actual code =)